wagtailmetadata/tags.py

```diff
@@ -27,8 +27,13 @@ def meta_tags(request, model):
 
     meta_image = model.get_meta_image()
     if meta_image:
+        meta_image_width = meta_image.width
+        meta_image_height = meta_image.height
         meta_image = get_meta_image_url(request, meta_image)
+    context['meta_image_width'] = meta_image_width
+    context['meta_image_height'] = meta_image_height
```

Comment:
If `meta_image` is None, this will throw a `NameError`. Just change this to:
```python
if meta_image:
    context['meta_image_width'] = meta_image.width
    context['meta_image_height'] = meta_image.height
    meta_image = get_meta_image_url(request, meta_image)

context['meta_image'] = meta_image
```